tests/e2e/script_test.go

```diff
@@ -251,6 +251,8 @@ func TestE2EScript(t *testing.T) {
 						// Allow some time for reconcile
 						// Maybe we should instead wait for observedState
 						time.Sleep(2 * time.Second)
```

Comment:
wait.Poll is a better pattern that a plain time.Sleep. It allows us to wait longer in worst case, less in the normal case and have a more descriptive error about what went wrong.